src/client/analytics/PlaytestData.tsx

```diff
@@ -62,7 +57,7 @@ const PlaytestData: React.FC<PlaytestDataProps> = ({ cubeAnalytics }) => {
         .filter(([oracle]) => cardDict[oracle])
         .map(([oracle, { elo, mainboards, sideboards, picks, passes }]) => ({
           card: {
-            exportValue: oracle,
+            exportValue: cardName(cardDict[oracle]),
```

Comment:
Functional change, use the cardName as the export value. The card is an object in order for renderCardLink to display the name and autocard hover, which means we also need the export value.